examples/albef/data/vqa_dataset.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import json
+import os
+import re
+from typing import List, Tuple, Union
+
+from examples.albef.data.transforms import ALBEFTransform
+
+from PIL import Image
+from torch import Tensor
+from torch.utils.data import Dataset
+
+
+class VQADataset(Dataset):
```

Comment:
@ebsmothers @katrina433   Is this the same dataset used by ALBEF pretraining? The dataset should be the same dataset used to train the model, right? 

cc @sophiazhi

Reply:
ALBEF pretraining data is in the format of `[{'image': path_of_image, 'caption': text_of_image}]`, where as vqa data is in the format of `[{'question_id': question_id, 'question': text_of_question, 'answer': text or list of text, 'image': path_of_image, 'dataset': 'vqa' or 'vg'}]`